src/accidental.ts

```diff
@@ -427,6 +427,17 @@ export class Accidental extends Modifier {
           // modified the accidental state
           const previouslyModified = modifiedPitches.indexOf(pitch) > -1;
 
+          // Remove accidentals
+          note.getModifiers().forEach(function (modifier, index) {
```

Comment:
Not sure what's going on here. Why do we need this block?

Reply:
This is required when the notes already include the accidentals before calling `applyAccidentals` (ie when they are created with `EasyScore`. I have included an additional test with EasyScore in accidentals. If this block is remved, the result is:

![image](https://user-images.githubusercontent.com/22865285/130578403-5bc3e3c5-017f-4c44-a163-74c224d3696e.png)